nsj_rest_lib/service/service_base.py

```diff
@@ -88,12 +89,15 @@ def _convert_to_entity_field(self, field: str) -> str:
 
         return entity_field_name
 
-    def _create_entity_filters(self, filters: Dict[str, Any]) -> Dict[str, List[Filter]]:
+    def _create_entity_filters(self, filters: Dict[str, Any], for_search: bool = False) -> Dict[str, List[Filter]]:
```

Comment:
Não entendi esse for_search...

Reply:
O tipo do filtro é definido pelo programador (no decorator), logo, não precisa de um parâmetro para mudar o tipo...

Reply:
isso foi uma saída para eu não ter que colocar esse lógica toda na minha api de novo, basicamente quando eu quiser sar o campo de busca em multiplos campos eu vou precisar que seja um like diferentemente de quando for filtrar algo que deverá ser equals, pelo que eu entendi na api vc implementa esses filtros diretamente no DTO então não consigo mudar isso dinamicamente